src/common.hh

```diff
@@ -60,24 +60,22 @@ struct ThreadPool final
     ThreadPool(size_t n_threads, std::function<void(const std::string&)> err);
     ~ThreadPool() noexcept;
 
-    void start();
     void push_to_job_queue(JobT&& job);
     void await_stop() noexcept;
 
   private:
     std::function<void(const std::string&)> error_handler_;
 
-    size_t n_threads_;
     std::vector<std::thread> threads_;
     mutable std::mutex jobs_mutex_;
     std::condition_variable cv_;
     std::queue<JobT> jobs_;
 
-    bool started_;
-    std::atomic<bool> should_stop_;
+    std::atomic<bool> is_accepting_jobs_;
 
     /// Multithreading
```

Comment:
Optional

This comment is not very informative. I think it's telling me that these methods are expected be executed on the worker threads this pool owns (and not on the client/calling thread). The other useful information is that the caller of these methods is expected to lock the require mutex to guarantee thread safety and consistency. So maybe update this along those lines?

Reply:
I'll just remove the comment.